common/azure-common/src/main/java/com/azure/common/exception/DecodeException.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.exception;
+
+/**
+ * Error raised during response deserialization.
+ */
+public class DecodeException extends AzureException {
```

Comment:
Can you clarify why this isn't a subclass of ServiceResponseException?

Reply:
Synced with Python team. We think it is a subset of ServiceResponseException, so moved to ServiceResponseException.